### DISCUSSION TITLE
[12주차] 온라인 스터디 (면접)

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#81] [General]
### 진행순서
~ 20:15 카훗`Kahoot`으로 입풀기
~ 21:00 1:N 모의면접 진행
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 🙂
* 좋았던 점
* 건의사항
* 느낀점

**Reply by ChaerinYu** [reply]
오늘의 참석자
@ChaerinYu @Kodo92 @leeejuhyeong

**Comment by doho-ho** [comment]
# 면접 질문
- Spring MVC의 작동 방식
- Error와 Exception의 차이
- Exception 처리는 어떻게 하는지? 
- Spring dependency injection 방법 중 선호하는 방법
- 트랜잭션의 특징 (ACID)

**Comment by leeejuhyeong** [comment]
### 준비(공부)해야할 질문
- 주로 사용하는 Exception 처리 방법
- 트랜잭션의 특징 정확하게 알고있기 (ACID)
